vsphere/vm-http-server/variables.tf

```diff
@@ -0,0 +1,7 @@
+variable "vm_name" {
+  type    = string
+}
+
+variable "http_server_port" {
```

Comment:
I would suggest to add a description to those variables, and if you really want to be very specific you can also add validation to the port using this https://www.terraform.io/docs/configuration/variables.html#custom-validation-rules.

Reply:
https://github.com/env0/templates/pull/18/commits/7abc286b2a8173563adaa410aa7c14df3d73c54b